test/integration/taxon_browsing_test.rb

```diff
@@ -248,10 +266,40 @@ def tagged_organisations
     ]
   end
 
+  def tagged_content_for_services
+    @tagged_content_for_services ||= generate_search_results(5, "services")
+  end
+
+  def tagged_content_for_guides
+    @tagged_content_for_guides ||= generate_search_results(5, "guides")
```

Comment:
Does this work? `generate_search_results` takes a supergroup - 'guides' isn't a supergroup?

Reply:
Just realised you've change the way `generate_search_results` works, but I still think it's misleading to be passing "guides" in here in the same way you're passing supergroups.